Tests/SwiftLintFrameworkTests/EmptyStringTests.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
You don't need to include this test file. The two cases listed here are the same from `triggeringExamples`. SwiftLint builds tests for snippets included on `nonTriggeringExamples` and `triggeringExamples` and automatically runs them to make sure the rule does what it says it does.